processor/src/main/java/org/mapstruct/ap/internal/processor/MapperCreationProcessor.java

```diff
@@ -74,6 +71,10 @@
 import org.mapstruct.ap.internal.util.Strings;
 import org.mapstruct.ap.internal.version.VersionInformation;
 
+import static org.mapstruct.ap.internal.prism.MappingInheritanceStrategyPrism.AUTO_INHERIT_FROM_CONFIG;
```

Comment:
They are removed from there due to the IntelliJ code style settings moving them at the end. I always try to not commit changes like this, but some are creeping through my checks 😄. Will remove it in the next commit.